CHANGELOG.md

```diff
@@ -4,6 +4,74 @@
 
 <!-- next version -->
 
+## v0.21.0
+
+This release includes version v0.116.0 and v0.117.0 of the upstream Collector components.
```

Comment:
```suggestion
This release includes version v0.117.0 of the upstream Collector components.
```

Reply:
Only one version of the components is actually included at a time. By adding both changelogs I think it's clear to users that they would need to look into both to see what has changed since our last release.